packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -172,7 +167,7 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 
 	private _options: ConductorOptions
 
-	private devices = new Map<string, BaseRemoteDeviceIntegration<DeviceOptionsBase<any>>>()
+	public readonly connectionManager = new ConnectionManager()
```

Comment:
should this be public, or should select methods be re-exposed on this conductor?